apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -231,4 +231,6 @@ public class ComponentsDefine {
 
     public static final OfficialComponent IMPALA_JDBC_DRIVER = new OfficialComponent(133, "Impala-jdbc-driver");
 
+    public static final OfficialComponent EVENT_MESH = new OfficialComponent(134, "EventMesh");
```

Comment:
This ID should be registered in the main repository first.

Reply:
https://github.com/apache/skywalking/blob/6984bbc65a4c12009e095e376b5e6ffb3a90db59/oap-server/server-starter/src/main/resources/component-libraries.yml#L434-L436

134 had been registered as Impala server, this is why we required for pre-registration before agent plugin PR.